documentation/modules/proc-downloading-product.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-product-downloads-{context}']
+= Downloading {ProductName}
+
+Download a ZIP file that contains the resources required for installation along with examples for configuration.
+
+.Prerequisites
+
+* Access to {ReleaseDownload}
+
+.Procedure
+
+. Download the `strimzi-x.y.z.zip` file from {ReleaseDownload}.
+
+. Unzip the file to any destination.
+* On Windows or Mac, you can extract the contents of the ZIP archive by double clicking on the ZIP file.
+* On Linux, open a terminal window in the target machine and navigate to where the ZIP file was downloaded.
++
+Extract the ZIP file by executing the following command:
++
+[source, shell, subs=+quotes]
+----
+unzip strimzi-xyz.zip
```

Comment:
Does windows have `unzip`? (It's such a long time since I used windows the answer might well be "yes")

Reply:
Don't think so. This is related to the second bullet, RHEL